mflowgen/common/custom-power-sram/outputs/power-strategy-dualmesh.tcl

```
@@ -0,0 +1 @@
+../../power-scripts/srams-power-strategy-dualmesh.tcl
```

Same with this?

---

Shouldn't this also be called something like leaf-with-srams-power-strategy?

---

I think combining the two leaves is a good idea .. it should be possible to skip the M5 power straps if there are no SRAMs in order to avoid the errors, right?

---

I think if you do `get_cells -quiet` it will suppress the errors.

---

Oh nice. I'll try it with `-quiet` and combine them